applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,1770 @@
+{
```

Comment:
<p>These are small things, but should there be commas:</p><ul><li>here in "perceived temperature, as they together "</li><li>between "high-risk, high-exposure"</li><li>after "However"</li><li>and after "In this notebook"?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Oh also, there's a double word "temperature temperature" in there.</p>